weTranslate/SearchHeaderViewController.swift

```diff
@@ -32,11 +48,74 @@ final class SearchHeaderViewController: UIViewController {
 
     // MARK: - UIViewController
 
-    override func loadView() {
-        self.view = SearchHeaderView(frame: .zero)
-    }
-
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        searchHeaderView.searchTextField.delegate = self
+        searchHeaderView.languagesPickerView.delegate = self
+        searchHeaderView.fromLanguageButton.addTarget(self, action: "selectLanguage:", forControlEvents: .TouchUpInside)
+        searchHeaderView.toLanguageButton.addTarget(self, action: "selectLanguage:", forControlEvents: .TouchUpInside)
+        searchHeaderView.swapButton.addTarget(self, action: "swap:", forControlEvents: .TouchUpInside)
+
+        view.addSubview(searchHeaderView)
+
+        searchHeaderView.leadingAnchor.constraintEqualToAnchor(view.leadingAnchor).active = true
+        searchHeaderView.trailingAnchor.constraintEqualToAnchor(view.trailingAnchor).active = true
+        searchHeaderView.topAnchor.constraintEqualToAnchor(view.topAnchor).active = true
+        searchHeaderView.bottomAnchor.constraintEqualToAnchor(view.bottomAnchor).active = true
```

Comment:
See https://github.com/jenniferlim/weTranslate/issues/9, will do that in a separate PR.